### DISCUSSION TITLE
Are pre-announced in-game events Out of Character?

### DISCUSSION THREAD

**Discussion by kovitikus** [evennia/evennia#2437] [General Discussion]
In the [MUD I played](https://www.eternalcitygame.com/index.php/community/eternal-city-game-world/), you were always encouraged to act according to your character's personality and also only behave as if your character only knows what they are currently capable of seeing, hearing, or otherwise perceiving.

But at the same time, the administration would post notices of upcoming in game events, naturally enticing players to show up to locations on characters that would have no reason to be there or knowledge that something is going to happen.

I wonder to myself if events should only be dynamic, and not announced. Thoughts on this conundrum?

**Comment by mrseeker** [comment]
If it's a notice board, I would say it's an in-game thing. You walk by there and read the pamflet. If it's like a messaging system, it's an OOC for sure.
Unless the admin has a way of mentally transferring that message to you, anything that cannot explained from a world's perspective is OOC.

**Reply by kovitikus** [reply]
Thanks for the reply. I feel like there's so many ways to tackle in-game in-character announcements. A message office with a notice board or a town crier, magic, technology. Pretty much any method of making an effort to spread news and information within the game to the appropriate people should be explored before simply dumping posts on a forum.

**Reply by Griatch** [reply]
Yes, if in-character is forefront, then the notice board can literally be something in-game too. I've played games where you had to go to the inn to read the board to see what in-game stuff was going on. Easy enough since the game was located mainly in one city. This was also a good excuse to meet people. The only 'meta' thing about it was that every notice board in the game-world had the same contents :) 

In more civilized game worlds, sending in-game mail is also nice. If you want people to come to your grand ball, maybe you send an invite to a guildmaster and ask them to in-turn distribute the invite to their guild members etc.
